woocommerce/woocommerce-android
# Issue/4558 level3 make create customer call

Resolves #4558 

**Merge after https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2086**

The PR adds create customer calls just before creating a payment intent. The resulting customer_id is added to the intent. "Loading" state is already shown at the moment I perform the call.

https://user-images.githubusercontent.com/4923871/128342093-59446bdd-01bc-442d-bf2a-b2e950729a23.mp4

### How to test
There is no UI facing changes.

1. Make a payment
2. Go to the stripes dashboard
3. Find your transaction
4. Scroll down and find JSON which represents the call to create a payment intent
5. Notice customer is set

![image](https://user-images.githubusercontent.com/4923871/128348757-b6514268-8825-4a03-992d-c7549ed749d9.png)
![image](https://user-images.githubusercontent.com/4923871/128348702-355bbb72-b070-48e3-ba08-4c8dbb6c0cef.png)


Update release notes:
